.github/CHANGELOG.md

```diff
@@ -1,10 +1,29 @@
 ## Release 0.2.0 (development release)
 
+### New features since last release
+
+### Improvements
+
+* The maximum wire value used in a statement within a definition will be the number of wires
+  declared. E.g., if, within a definition, a single statement is applied on wire 4, then the
+  declared wires will be 0, 1, 2, 3, and 4.
```

Comment:
Minor point, but the capitalization of "e.g." feels odd. We can rephrase the sentence to avoid this:
```suggestion
  declared. For example, if a single statement within a definition is applied to wire 4, then the
  declared wires will be 0, 1, 2, 3, and 4.
```

Reply:
I actually tend to agree here. Will update!